tenzir.yaml.example

```diff
@@ -18,6 +18,14 @@ tenzir:
   # The file system path used for persistent state.
   db-directory: "tenzir.db"
 
+  # The file system path used for persistent state.
+  # Defaults to one of the following paths, selecting the first that is
+  # available:
+  #   - $XDG_CACHE_HOME
+  #   - $XDG_HOME_DIR/.cache/tenzir or
```

Comment:
- If XDG_HOME_DIR is not set, you should probably use HOME instead.
- On macOS, caches should go under `$HOME/Library/Caches/tenzir` instead, unless either of the XDG variables are defined.

Reply:
Are the `XDG_` variables usually defined on mac, or would that be the exception?

Reply:
They are undefined, some people define them because some software respects it, but it's far from standard behavior.